docker-compose.dev.yml

```
@@ -0,0 +1,33 @@
+        build:
+            context: .
+            dockerfile: Dockerfile.postgres
+        environment:
+            - POSTGRES_PASSWORD=musicbrainz
+        ports:
+            - "5432:5432"
+        volumes:
+            - db_data:/var/lib/postgresql/data
+        shm_size: 8gb
+        healthcheck:
+            test: [ "CMD-SHELL", "pg_isready -U musicbrainz -d musicbrainz" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)